source/dpp/runtime/options.d

```diff
@@ -46,23 +47,15 @@ struct Options {
         else
             enforce(args.length >= 2, "Not enough arguments\n" ~ usage);
 
+        args = response_expand(args);
+
         dppFileNames = args.filter!(a => a.extension == ".dpp").array;
-        enforce(dppFileNames.length != 0, "No .dpp input file specified\n" ~ usage);
```

Comment:
This is where I meant to write this:

no need to have this, if the target compiler can handle what's been given then we just pass it through.